recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    - gvconfig_libdir.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - cairo  # [linux]
+    - expat  # [linux]
```

Comment:
>The linker was complaining that it was unable to find the expat dynamic library.

I believe the error you're referring to is this one, which occurs during `make install`:

```
Making install in dot
 ../../config/install-sh -c -d '/miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin'
  /bin/sh ../../libtool   --mode=install /usr/bin/install -c dot dot_builtins '/miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin'
libtool: install: /usr/bin/install -c .libs/dot /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot
libtool: install: /usr/bin/install -c .libs/dot_builtins /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot_builtins
/Applications/Xcode.app/Contents/Developer/usr/bin/make  install-exec-hook
(cd /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin; if test -x dot; then for i in neato twopi fdp circo osage patchwork sfdp; do rm -f $i; ln -s dot $i; done; fi;)
if test "x" = "x"; then if test -x /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot; then if test -x /sbin/ldconfig; then /sbin/ldconfig 2>/dev/null; fi; /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot -c; else /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot_static -c; fi; fi
dyld: Library not loaded: @rpath/libexpat.1.dylib
  Referenced from: /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot
  Reason: Incompatible library version: dot requires version 8.0.0 or later, but libexpat.1.dylib provides version 7.0.0
/bin/sh: line 1: 94089 Trace/BPT trap: 5       /miniconda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/bin/dot -c
```

>I really don't know why, but since Homebrew doesn't require it, I just removed it from OS X.

I think Homebrew doesn't require it because it is apparently already installed on OS X in a system location: `/usr/lib/libexpat.1.dylib`.  Well, that's where it lives if you have the XCode command-line tools installed.  If you don't have the XCode command-line tools, I'm not sure where it lives (if it's present at all).

If we would prefer to use conda for `expat` instead of the system, we can fix the above-mentioned error with an extra command after `make`, but before `make install`.  If you think that change is worth making, I've submitted a PR-onto-this-PR:

https://github.com/ccordoba12/staged-recipes/pull/2

If you don't think it's worth it, then just ignore that PR :-)